tests/local_wikibase/README.md

```diff
@@ -30,24 +30,6 @@ http://wdqs-frontend.localhost/
 
 http://quickstatements.localhost/
 
-### Programmatic Access
```

Comment:
what's the reason for removing these docs? do the docs need more attention?

Reply:
deleted because this was the only place that the get_all_properties() method was actually used! could replace this with a different example if you think it's needed?

Reply:
it could be worth replacing with something like "you can access the test instance using the `WikibaseSession` class", with a link to the implementation?

or just leave it - i feel like you have a much better sense of who will use which docs in this repo than I do! (and there are lots of great docs, and well-documented code, already)